setup/pom.xml

```diff
@@ -64,8 +63,8 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
 				<configuration>
-					<source>11</source>
-					<target>11</target>
+					<source>17</source>
```

Comment:
Da sollte man wohl das Dockerfile auch anpassen.

Reply:
In der Tat.